thirdparty/paho.mqtt.c/.travis.yml

```diff
@@ -0,0 +1,47 @@
+sudo: true
```

Comment:
this file doesn't seem like it's needed.

Reply:
will remove